content/en/network_monitoring/dns/_index.md

```diff
@@ -22,20 +22,23 @@ further_reading:
 
 {{< img src="network_performance_monitoring/dns_default.png" alt="DNS Monitoring" >}}
 
+<div class="alert alert-info">
+Upgrade to agent version 7.33 to enable DNS monitoring.
+</div>
+
 DNS Monitoring provides an overview of DNS server performance to help you identify server-side and client-side DNS issues. By collecting and displaying flow-level DNS metrics, this page can be used to identify:
 
 * The pods or services making DNS requests and the servers receiving those requests.
 * The endpoints making the most requests or making requests at the highest rate.
 * If a DNS server’s response time to requests has gradually or suddenly increased.
-* The DNS servers with a high error rate and the type of errors being emitted. 
+* The DNS servers with a high error rate and the type of errors being emitted.
+* Which domains are being resolved.
 
 ## Setup
 
 Before you can begin to use DNS Monitoring, [set up Network Performance Monitoring][1]. Also ensure you are using the latest version of the Agent, or at least Agent v7.23+ for Linux OS, and v7.28+ for Windows Server. Once installed, a ‘DNS’ tab is accessible in the Network Performance Monitoring product.
 
-If you are using [Kubernetes on Helm][2], to enable DNS monitoring, you need to set `collectDNSStats:` to `true` in the [values.yaml][3].
-
-Are you looking for Network Device Monitoring instead? See the [NPM setup instructions][4].
+Are you looking for Network Device Monitoring instead? See the [NPM setup instructions][2].
```

Comment:
I know this is not part of changes being made in this PR -  should it be `NDM` instead of `NPM`?